fv3gfs-physics/tests/savepoint/conftest.py

```diff
@@ -99,12 +111,32 @@ def read_serialized_data(serializer, savepoint, variable):
     return data
 
 
-def process_grid_savepoint(serializer, grid_savepoint, rank, *, backend: str):
-    grid = make_grid(grid_savepoint, serializer, rank, backend=backend)
-    fv3core._config.set_grid(grid)
+def Grid(serializer, grid_savepoint, rank, layout, *, backend: str):
```

Comment:
Please use a lowercase name, since this isn't returning a class.

Reply:
Changed to `upper_grid`